src/main/webapp/user-private-comments.jsp

```diff
@@ -16,20 +18,20 @@
   <%if (currUser) {%>
     <h1 id="pc-header">Your Private Task Comments:</h1>
     <%for (TaskData task : tasks) {%>
-      <li class="task">
-        <%if (task.getStatus().name().equals("COMPLETE")) {%>
-          <button class="complete" type="button" onclick="location.href='task?taskID=<%=task.getTaskID()%>'">
-        <%} else if (task.getStatus().name().equals("INCOMPLETE")){%>
-          <button class="incomplete" type="button" onclick="location.href='task?taskID=<%=task.getTaskID()%>'">
-        <%}%>
+      <li class="task-with-pc">
+        <button class="inline deep-button" type="button" onclick="location.href='task?taskID=<%=task.getTaskID()%>'">
           <%=task.getName()%>
         </button>
+        <%request.setAttribute("task", task);%>
+        <%request.setAttribute("clickable", false);%>
+        <jsp:include page="task-status-checkmark.jsp"/>
+        <p class="inline"><%=task.getDescription()%></p>
         <form action="/user-profile" method="POST">
           <input type="hidden" name="taskID" value="<%=task.getTaskID()%>">
           <br>
           <textarea style="width:300px; height:100px;" type="text" name="message-<%=task.getTaskID()%>" rows="6" cols="50"><%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim()%></textarea>
           <br>
-          <button class="deep-button" type="submit" class="deep-button">Update</button>
+          <button class="deep-button" style="font-size: 12px;" type="submit" class="deep-button">Update Comment</button>
```

Comment:
Please move into the css for consistency.  If it helps, you can have 2 classes in the same tag 
eg `class="deep-button submit-button"` 

then reference them in the css file separately like:
```
.deep-button{...}
.submit-button{
  font-size:  12px;
}
```